src/status_im/navigation/screens.cljs

```diff
@@ -413,6 +413,9 @@
      :options   {:insets {:top? true :bottom? true}}
      :component wallet-keypair-name/view}
 
+    {:name      :screen/wallet.recovery-phrase
```

Comment:
it's a nit, but should we not have these consistent?
i.e 
` `` 
{:name      :screen/wallet.enter-seed-phrase
     :component enter-seed-phrase/view}
 ```
 or 
 ```
 {:name :screen/wallet.recovery-phrase
 :component recovery-phrase/view}